docs/source/api_reference/utils.rst

```diff
@@ -384,6 +384,10 @@ The utilites here are used to create an object that contains information about a
 
 .. autofunction:: deepchem.utils.differentiation_utils.assert_runtime
 
+.. autofunction:: deepchem.utils.dft_utils.hamilton.intor.utils.np2ctypes
```

Comment:
Add a subsection here and explain the purpose of these functions and how they wrap around a C code